src/components/map/mapactions/custom-map.js

```diff
@@ -118,22 +118,28 @@ export class VlCustomMap extends VlMapWithActions {
     this.baseLayers.push(baseLayer);
 
     if (this.overviewMapControl) {
+
       this.overviewMapControl
         .getOverviewMap()
         .getLayers()
         .getArray()
         .push(overlayMapLayer);
+
+      // set second layer as next in overview control
+      this.overviewMapControl
```

Comment:
Zet de tweede layer in de overviewcontrol als layer die als volgende aangeklikt kan worden.